### PR TITLE
[ Annesa ] Admin Dashboard Roster

### PR DESCRIPTION
**Pull request by annesatran**
- Populated the Admin Dashboard's Roster: displays youth with "active" accounts in one downloadable table and youths with "inactive" accounts in another downloadable table. The former indicates youth in the current summit and the other indicates youth who participated in previous summits.

  ```
  AdminRoster
    L RosterTable
         L DownloadCsvBtn
  ```

### CODE REVIEW

File: src/app/(dashboard)/admin/roster/DownloadCsvBtn.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { MdDownload } from "react-icons/md";
+
+function handleCsvDownload(id, fileName, roster) {
```
Comment (carolinebnguyen) [inline src/app/(dashboard)/admin/roster/DownloadCsvBtn.js:5]: The download works well! 

Nit: In the .csv file, there's extra column at the end without a header. Is it possible for you to remove that?

File: src/app/(dashboard)/admin/roster/page.js [hunk 2]
```diff
@@ -1,3 +1,139 @@
...
+      <div className="join-item flex flex-row justify-between">
+        <div>
+          <h3 className="text-lg font-bold">{title}</h3>
+          <div className="text-xs font-medium opacity-70">{roster.length} entries</div>
+        </div>
+        <div className="mt-1">
+          <DownloadCsvBtn id={downloadId} fileName={fileName} roster={roster}/>
+        </div>
+      </div>
+      <div className="overflow-x-scroll max-h-screen">
```
Comment (carolinebnguyen) [inline src/app/(dashboard)/admin/roster/page.js:71]: Nit: We have to scroll down the page to access the horizontal scrollbar for the table. Is it possible to reduce the height of the table or to make the horizontal scrollbar stay visible at the bottom of the window?

**Review by carolinebnguyen — APPROVED** [review]
Looks good! Just some minor things that I noticed, but this is amazing 😄

File: src/app/(dashboard)/admin/roster/DownloadCsvBtn.js (hunk 1 above)
Comment (annesatran) [reply src/app/(dashboard)/admin/roster/DownloadCsvBtn.js:5]: Nice catch! Resolved now.

File: src/app/(dashboard)/admin/roster/page.js (hunk 2 above)
Comment (annesatran) [reply src/app/(dashboard)/admin/roster/page.js]: Thanks for the suggestion! I ended up making the height of the table shorter.

File: src/app/(dashboard)/admin/roster/DownloadCsvBtn.js (hunk 1 above)
Comment (carolinebnguyen) [reply src/app/(dashboard)/admin/roster/DownloadCsvBtn.js:5]: Perfect, looks great!

**Review by carolinebnguyen — APPROVED** [review]
Go Annesa!! 🎉

**Pull request merged**